### PR TITLE
Use all declared namespaces to not assume "vs".

### LINKED ISSUE #155: CapabilitiesWriter always assumes "vs" prefix
TAP implementations with Examples interfaces for the Browser (i.e. `vr:WebBrowser`) are being converted to `vs:WebBrowser` in the `CapabilitiesWriter`.  The resulting document is no longer correct.

### PR DESCRIPTION
**Pull request by at88mph**
Closes #155

### CODE REVIEW

File: cadc-registry/src/main/java/ca/nrc/cadc/reg/CapabilitiesWriter.java [hunk 1]
```diff
@@ -95,6 +95,7 @@ public class CapabilitiesWriter {
 
     Namespace vosi = Namespace.getNamespace("vosi", "http://www.ivoa.net/xml/VOSICapabilities/v1.0");
     Namespace vs = Namespace.getNamespace("vs", "http://www.ivoa.net/xml/VODataService/v1.1");
+    Namespace vr = Namespace.getNamespace("vr", "http://www.ivoa.net/xml/VOResource/v1.0");
```
Comment (pdowler) [inline cadc-registry/src/main/java/ca/nrc/cadc/reg/CapabilitiesWriter.java:98]: XMLConstants has all the namespace URIs so please update to use those (including the other hard-coded ones above).

File: cadc-registry/build.gradle [hunk 2]
```diff
@@ -16,7 +16,7 @@ sourceCompatibility = 1.8
 
 group = 'org.opencadc'
 
-version = '1.6.1'
+version = '1.6.2'
```
Comment (pdowler) [inline cadc-registry/build.gradle:19]: this now 1.6.3

**Review by pdowler — CHANGES_REQUESTED** [review]
VODataService is an extension of VOResource so it might not be technically wrong (only cosmetic) that the namespace prefix is vs instead of vr. However, there is a documented practice in the ivoa to stick to recognizable prefixes (even though it isn't technically required) so we should try to fix that if it applies here.

This needs to be tested by adding a vs:WebBrowser capability to the sample-capabilities.xml file and making sure it is schema-valid.

Also some "technical debt" items below. I'll look at the remaining changes after the test is augmented and passes.

File: cadc-registry/src/main/java/ca/nrc/cadc/reg/CapabilitiesWriter.java (hunk 1 above)
Comment (at88mph) [reply cadc-registry/src/main/java/ca/nrc/cadc/reg/CapabilitiesWriter.java]: OK.  The `XMLConstants` doesn't have a prefix for all of the known Namespace URIs though.  I've only included the hardcoded ones from `CapabilitiesWriter` and the two that were already there.  Does that match up?

File: cadc-registry/src/main/java/ca/nrc/cadc/reg/XMLConstants.java [hunk 3]
```diff
@@ -149,5 +151,10 @@ public class XMLConstants {
 
     public static final Namespace CAPABILITIES_NS = Namespace.getNamespace("vosi", VOSICAPABILITIES_10_NS.toString());
     public static final Namespace VODATASERVICE_NS = Namespace.getNamespace("vod", VODATASERVICE_11_NS.toString());
+    public static final Namespace VODATASERVICE_VS_NS = Namespace.getNamespace("vs", VODATASERVICE_11_NS.toString());
+    public static final Namespace VORESOURCE_NS = Namespace.getNamespace("vr", VORESOURCE_10_NS.toString());
 
+    public static final List<Namespace> NAMESPACE_LIST =
```
Comment (pdowler) [inline cadc-registry/src/main/java/ca/nrc/cadc/reg/XMLConstants.java:157]: This NAMESPACE_LIST constant seems specific to a capabilities document so it probably belongs in CapabilitiesWriter directly. It is certainly a subset of all the NS constants declared here and included in the SCHEMA_MAP.

yeah, move to  CapabilitiesWriter

**Review by pdowler — CHANGES_REQUESTED** [review]

File: cadc-registry/src/main/java/ca/nrc/cadc/reg/XMLConstants.java (hunk 3 above)
Comment (at88mph) [reply cadc-registry/src/main/java/ca/nrc/cadc/reg/XMLConstants.java]: Done.

**Review by pdowler — APPROVED** [review]

**Pull request merged**